README.md

```diff
@@ -49,22 +50,22 @@ Or start by creating a workspace and cloning the template repository:
 
     ```bash
     cd ~/template_ws
-    source /opt/ros/<ROS_DISTRO>/setup.bash
+    source /opt/ros/humble/setup.bash
     sudo rosdep init
     rosdep update
-    rosdep install --from-paths src --ignore-src -r -i -y --rosdistro <ROS_DISTRO>
+    rosdep install --from-paths src --ignore-src -i -y
     ```
 
 1. Build the project
 
     ```bash
-    colcon build --cmake-args -DBUILD_TESTING=ON
+    colcon build
     ```
 
 1. Source the workspace
 
     ```bash
-    . ~/template_ws/install/setup.sh
+    source install/setup.bash
```

Comment:
```suggestion
    source <path_to_template_workspace>/install/setup.bash
```

Reply:
Why is this recommended? If you just built, the install directory is relative to the build directory that was just created.

My change matches the upstream tutorial: https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Colcon-Tutorial.html#source-the-environment

I've found new users to ROS use VM's and don't have copy-paste, so the less typing, the better.

Reply:
I was mirroring the previous comment. Yeah trying to look for some standards, ROS docs one helps, thanks. Discarding this suggestion